exec/internal/compile/backend_amd64.go

```diff
@@ -518,6 +544,13 @@ func (b *AMD64Backend) emitDivide(builder *asm.Builder, regs *dirtyRegs, op byte
 		ext.As = x86.ACQO
 		builder.AddInstruction(ext)
 		prog.As = x86.AIDIVQ
+	case ops.I32DivS, ops.I32RemS:
+		ext := builder.NewProg()
+		ext.As = x86.ACDQ
+		builder.AddInstruction(ext)
+		prog.As = x86.AIDIVL
+	case ops.I32DivU, ops.I32RemU:
+		prog.As = x86.ADIVL
```

Comment:
now that the number of ops increased, perhaps replace the `default` case with the actual values (`ops.I64{DivU,RemU}`) and add a panic-y catchall `default` case?